src/schema_generators/generators.cljc

```diff
@@ -223,3 +224,17 @@
   "Sample a single element of low to moderate size."
   [& generator-args]
   (generators/generate (apply generator generator-args) 10))
+
+(s/defn generate-alt :- s/Any
```

Comment:
This is a poor name choice. Any suggestions while I'm trying to come up with something better?

Reply:
I would do perhaps
`(s/defn generate-deterministic [size seed & generator-args] (generators/generate (apply generator generator-args) size seed))`

Although really I would prefer to just get out of this game entirely (and eventually remove `sample` and `generate` from this library entirely).  This is really a library about making generators, and it was probably a mistake to ever provide wrappers for using them rather than encouraging just using test.check directly.

Reply:
The rational of only handling generator creation logic in this lib sounds good. I'll implement these generation changes on my side